src/main/java/uk/ac/ebi/embl/api/validation/helper/MasterSourceFeatureUtils.java

```diff
@@ -122,9 +123,17 @@ public SourceFeature constructSourceFeature(SampleEntity sample, TaxonHelper tax
 			if (isCovidTaxId(sourceFeature.getTaxId()) && tag != null) {
 				// Master source qualifiers values created from multiple sample fields are constructed here.
 				if (tag.toLowerCase().contains("latitude")) {
-					latitude = value;
+					try {
```

Comment:
what if latitude and longitude value already has direction indicator (S,N or something)